release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -2773,3 +2772,24 @@ link:https://access.redhat.com/solutions/6902561[{product-title} 4.10.9 containe
 ==== Updating
 
 To update an existing {product-title} 4.10 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc[Updating a cluster within a minor version by using the CLI] for instructions.
+
+[id="ocp-4-10-10"]
+=== RHSA-2022:1357 - {product-title} 4.10.10 bug fix and security update
+
+Issued: 2022-04-20
+
+{product-title} release 4.10.10 is now available. The bug fixes that are included in the update are listed in the link:https://access.redhat.com/errata/RHSA-2022:1241[RHSA-2022:1357] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2022:1357[RHBA-2022:1357] advisory.
```

Comment:
First Errata link doesn't match text, and text is the same as the second link - wanna check these :)

Reply:
Whoops! I did this pre-coffee this morning. Updated!